tests/network-ovn

```diff
@@ -2127,6 +2127,57 @@ ovn_leases_tests() {
         lxc network delete lxdbr0
 }
 
+ovn_vlan_uplink_tests() {
+    echo "==> vlan uplink"
```

Comment:
Please change "vlan" to "VLAN" in end user messages as its an acronym.

Reply:
Done